### PR TITLE
Setup ci

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
Set up CI.

(very basic minimal test to catch regression)

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+var Lab = require('lab');
+var Code = require('code');
+
+var lab = exports.lab = Lab.script();
+var describe = lab.describe;
+var it = lab.it;
+var expect = Code.expect;
+
+var Fs = require('fs');
```
Comment (bevacqua) [inline test/test.js]: Please use lowercase here

File: test/test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+var Htmlterminal = require('..');
```
Comment (bevacqua) [inline test/test.js]: Please use lowercase here, you can call it just `hget` since that's what you're requiring

**Comment by bevacqua** [comment]
Great stuff. Please see my comments and I'll merge :)

**Comment by AdrieanKhisbe** [comment]
*et voilà* :smiley:

**Pull request merged**